CosmicReco/inc/PDFFit.hh

```diff
@@ -75,7 +75,7 @@ class GaussianDriftFit : public ROOT::Minuit2::FCNBase {
 
     GaussianDriftFit(ComboHitCollection _shs, StrawResponse const& _srep, const Tracker* _tracker) : shs(_shs), srep(_srep), tracker(_tracker){}; 
 
-    double Up() const { return 0.5;};
+    double Up() const { return 1.0;};
     double operator() (const std::vector<double> &x) const;
```

Comment:
What is this number?

Reply:
This just tells minuit that operator() returns a chi-squared and not a log likelihood for when it scales the variances

Reply:
Could you please add a comment to that effect?